docs/sources/guides/whats-new-in-v6-3.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "What's New in Grafana v6.3"
+description = "Feature & improvement highlights for Grafana v6.3"
+keywords = ["grafana", "new", "documentation", "6.3"]
+type = "docs"
+[menu.docs]
+name = "Version 6.3"
+identifier = "v6.3"
+parent = "whatsnew"
+weight = -14
++++
+
+# What's New in Grafana v6.3
+
+For all details please read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md)
+
+The main highlights are:
```

Comment:
@torkelo don't you want to include info about data links?

Reply:
completely forgot that, I did take a screenshot of it add added it to the website repo, but forgot to add it here :)

Reply:
Heh, no probs!